src/SqlToCsharp/ColumnInfo.cs

```diff
@@ -26,9 +26,10 @@ public ColumnInfo(ColumnDefinition c, IList<ConstraintDefinition> tableConstrain
         private static int? FindPrimaryKeyConstraint(ColumnDefinition column, IList<ConstraintDefinition> tableConstraints)
         {
             var cons =
-                from UniqueConstraintDefinition uc in tableConstraints
-                from col in uc.Columns.Indexed()
-                where col.item.Column.MultiPartIdentifier.Identifiers.Last().Value == column.ColumnIdentifier.Value
+				from uc in tableConstraints

```

Comment:
use `from us in tableConstraints.OfType<UniqueConstraintDefinition>`.

the `is` operator and the cast are duplication。